test/automation/src/sql/configurePythonDialog.ts

```diff
@@ -25,11 +25,14 @@ export class ConfigurePythonDialog extends Dialog {
 		const newPythonInstallation = '.modal .modal-body input[aria-label="New Python installation"]';
 		await this.code.waitAndClick(newPythonInstallation);
 
-		const nextButton = '.modal-dialog .modal-content .modal-footer .right-footer .footer-button a[aria-label="Next"][aria-disabled="false"]';
+		const pythonInstallLocationDropdownValue = `${dialog} select[aria-label="Python Install Location"] option`;
+		await this.code.waitForElement(pythonInstallLocationDropdownValue);
+
+		const nextButton = '.modal-dialog .modal-content .modal-footer .right-footer .footer-button a[aria-label="Next"][aria-disabled="false"]:not(.disabled)';
```

Comment:
We already have the aria-disabled=false selector so I'm curious what this is changing - are they not kept in sync?

Reply:
Looking more closely, they are kept in sync so this should not change anything..